### PR TITLE
Each PUT return an updated payload

### PR DESCRIPTION
**Pull request by yroffin**
No each PUT return an updated payload : 
- /v1/applications/:domain/:name/versions/:version
- /v1/applications/:domain/:name/versions/:version/badgeratings/:badgeslug
- /v1/environments/:slug
- /v1/badges/:slug

### CODE REVIEW

File: api/v1/type.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ type Release struct {
 	CreatedAt    time.Time       `json:"_createdAt" binding:"-"`
 	UpdatedAt    time.Time       `json:"_updatedAt" binding:"-"`
 	DeletedAt    *time.Time      `json:"-" binding:"-"`
-	BadgeRatings *postgres.Jsonb `json:"-" binding:"-"`
+	BadgeRatings *postgres.Jsonb `json:"badgeratings" binding:"-"`
```
Comment (rayene) [inline api/v1/type.go]: What is the reason behind this change ?
I remember we said that we should not show badge ratings on the app's resource.

Comment (yroffin) [reply api/v1/type.go]: Done
